pkg/profiles/tap.go

```diff
@@ -60,6 +61,10 @@ func tapToServiceProfile(client pb.ApiClient, tapReq *pb.TapByResourceRequest, n
 
 	tapClient, err := client.TapByResource(ctx, tapReq)
 	if err != nil {
+		if strings.HasSuffix(err.Error(), context.DeadlineExceeded.Error()) {
```

Comment:
is it possible to do a object equality comparison instead of a string comparison for this one?

Reply:
I think you have to do the string suffix comparison because the error that gets returned here (err) is `Post https://localhost:6443/api/v1/namespaces/linkerd/services/linkerd-controller-api:http/proxy/api/v1/TapByResource: context deadline exceeded` whereas the value of `context.DeadlineExceeded.Error()` is only `context deadline exceeded`